.nvmrc

```diff
@@ -1 +1 @@
-v14.8
+v14.15
```

Comment:
Are you using a feature 14.15 requires? I've been setting the .nvmrc file to the earliest supported version of Node. 14.8 is where top-level await was added so that's why I picked that one.

Reply:
I think that was from the branch I based it on (?)

https://github.com/redis/redis-om-node/commit/75026446e27c20fe6cd7829525b17935937bb090